pimmi/cli/__main__.py

```diff
@@ -315,6 +326,12 @@ def viz(clusters, output, **kwargs):
     from_clusters_to_viz(clusters, output)
 
 
+def download(dataset, data_dir, **kwargs):
+    dir = data_dir if data_dir else os.getcwd()
```

Comment:
The user would probably not like to download the folder in data-dir/demo_dataset/small_dataset but directly in data-dir